docs/api_v2/api_v2_reference.apib

```diff
@@ -1938,6 +1938,144 @@ In order to save new items to the dataset, send HTTP POST request with JSON payl
     + Attributes
         - data (Run, required)
 
+# Group Actor runs
+
+The API endpoints described in this section enable you to manage Apify actor run.
+
+## Run object [/v2/actor-runs/{runId}{?token,waitForFinish}]
+
+### Get run [GET]
+
+Gets an object that contains all the details about a specific run of an actor.
+
+By passing the optional `waitForFinish=1` parameter the API endpoint will synchronously wait for the run to finish.
+This is useful to avoid periodic polling when waiting for actor run to complete.
```

Comment:
Would it not be good to have Run Collection here as well? Just to have all things related to runs in the same place?
https://docs.apify.com/api/v2#/reference/actors/run-collection

Reply:
Same as comment above